server/src/main/java/org/spine3/server/aggregate/AggregateRepository.java

```diff
@@ -263,7 +263,7 @@ public void store(A aggregate) {
      * <p>If the aggregate is not available in the repository this method returns
      * a newly created aggregate.
      *
-     * <p>If the aggregate is “invisible” to regular queries, {@code Optional.absent()}
+     * <p>If the aggregate is "invisible" to regular queries, {@code Optional.absent()}
```

Comment:
We can specify encoding of the source files in Javadoc tool by specifying `-encoding` options. http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#encoding